parlai/agents/transformer/transformer.py

```diff
@@ -25,12 +25,12 @@
 def add_common_cmdline_args(argparser):
     argparser.add_argument('-esz', '--embedding-size', type=int, default=300,
                            help='Size of all embedding layers')
-    argparser.add_argument('--n-layers', type=int, default=2)
-    argparser.add_argument('--ffn-size', type=int, default=300,
+    argparser.add_argument('-nl', '--n-layers', type=int, default=2)
+    argparser.add_argument('-hid', '--ffn-size', type=int, default=300,
                            help='Hidden size of the FFN layers')
     argparser.add_argument('--attention-dropout', type=float, default=0.0)
     argparser.add_argument('--relu-dropout', type=float, default=0.0)
-    argparser.add_argument('--n-heads', type=int, default=3)
+    argparser.add_argument('--n-heads', type=int, default=2)
```

Comment:
Although I am pretty sure here it doesn't change anything, changing default parameters seem like a bad practice since some people may have run experiments while letting the parameter blank. Saved somewhere the command line to reproduce and now it doesn't.

Reply:
It does have a BETA warning at the top. Saved models will still produce the exact same results.

If you ask, I will move it back.

Reply:
Right. Didn't see the beta. Seems legit.